pages/en/lb3/LoopBack-types.md

```diff
@@ -65,6 +65,17 @@ The following table summarizes LoopBack's data types.
       </td>
     </tr>
     <tr>
+      <td>DateString</td>
+      <td>
+        <p>LoopBack<a href="http://apidocs.strongloop.com/loopback-datasource-juggler/#datestring" class="external-link" rel="nofollow"> DateString</a> object</p>
```

Comment:
I assume that's what @kjdelisle is talking about re https://github.com/strongloop/loopback-datasource-juggler/pull/1356.  Once that lands (with some modifications) _and_ juggler is published to npm, then the link should work.